src/styles/blocks/card.scss

```diff
@@ -0,0 +1,126 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+$white: #fff;
+$concrete-color: #f3f3f3;
```

Comment:
if you want your code be even more reusable you can keep those variables in another file, outside blocks - like `utils/variables`